example-hotel-reservation/README.md

```diff
@@ -1,3 +1,554 @@
 # Booking Platform
 
 This example shows you how to make a reservation in a hotel by taking all the profit of webviews and custom messages.
+
+
+**What's in this document?**
+
+- [How to use this example](#how-to-use-this-example)
+- [Webchat components](#webchat-components)
+  - [1. Cover Component](#1-cover-component)
+  - [2. Custom Messages](#2-custom-messages)
+    - [2.1 Custom Message from bot](#21-custom-message-from-bot)
+    - [2.2 Custom Message from user](#22-custom-message-from-user)
+  - [3. Persistent Menu](#3-persistent-menu)
+- [Carousel](#carousel)
+- [Webviews](#webviews)
+
+
+## How to use this example
+
+1. From your command line, download the example by running:
+   ```bash
+   $ botonic new <botName> booking-platform
+   ```
+2. `cd` into `<botName>` directory that has been created.
+3. Run `botonic serve` to test it in your local machine.
+
+## Webchat components
+
+In this section, we will see the most relevant webchat components that have been used to create this example.
+
+
+#### 1. Cover Component
+
+This is the first component that will appear when a user opens the webchat. It's implemented in `src/webchat/cover-component.js`.
+
+We have used  `styled-components` that allows us to define new React component with our styles attached to it.
+
```

Comment:
i would add here the name of the file just before the code snippet (for all code snippets). between **** like here https://github.com/hubtype/botonic-examples/tree/master/example-nlu#22-creating-your-own-custom-model